requirements.txt

```diff
@@ -1,3 +1,4 @@
 requests
 
 -e .
++e git+https://github.com/edx/xblock-utils.git#egg=xblock-utils
```

Comment:
It looks like this code is AGPL3 licensed. Is it possible to avoid using this library to avoid the licensing issues around mixing AGPL3 code in a non-AGPL3 project?